src/tree_data_helpers.c

```diff
@@ -568,38 +570,26 @@ ly_store_prefix_data(const struct ly_ctx *ctx, const char *value, size_t value_l
             prefixes = *prefix_data_p;
         }
 
+        value_end = value + value_len;
```

Comment:
I would switch this line and the next one, to signify that this line is directly connected to the loop and functionality in it, it is not separate.

Reply:
ok